index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+            rel="stylesheet"
+            href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+        />
+        <link
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet"
+        />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap"
+            rel="stylesheet"
+        />
+        <link rel="stylesheet" href="styles.css" />
+        <title>Contributing to Open Source Part I: The Easy Way</title>
+    </head>
+    <body>
+        <div class="content">
+            <nav class="nav">
+                <div class="nav-left-logo">
+                    <button class="icon-only-button">
+                        <i class="fa fa-bars icon-color" aria-hidden="true"></i>
+                    </button>
+                    <img
+                        class="logo"
+                        src="https://archive.vn/Bss88/51a3832e6af6792add876216da444db23a4bea96.png"
+                        alt="Logo"
+                    />
+                </div>
+                <div class="search">
+                    <input type="text" placeholder="Search" />
+                    <button type="submit">
```

Comment:
`<button></button>` has a `type="submit"` by default.
```suggestion
                    <button>
```